### ISSUE TITLE
 I have tried to edit media,Its returning “True” in response, But category is not updated yet.

### ISSUE THREAD

**Issue by inder243** [brianhorakh/vita#14]
: http://www.vitatheapp.com/vita-api/EventMediaAnnotate?evid=B49BD75E-0889-11E6-B39A-D99FB462889C&authid=4F45325C-F71C-11E5-A06D-A7AA58C3F44C&uid=101&mediaid=256&category=update-other&title=testingeventmedia&type=flag&reason=updating%20media

Its returning “True” in response, But category is not updated yet.
http://www.vitatheapp.com/vita-api/EventList?my-uid=101

**Comment by brianhorakh** [comment]
It is not possible to update the category of an Event from EventMediaAnnotate please use EventUpdate.

EventUpdate?type=update-category&category=xxx

**Comment by inder243** [comment]
http://www.vitatheapp.com/vita-api/EventMediaUpdate?evid=D3347DC6-F71B-11E5-91E5-A7AA58C3F44C&authid=4F45325C-F71C-11E5-A06D-A7AA58C3F44C&mediaid=327&uid=101&title=updatedtitle&caption=updatedcaption

 Response : {"success":false,"error_msg":"internal server error Undefined subroutine &VITA::update called at /httpd/vita/perllib/VITA.pm line 1595.\n","now":"2016-06-06 05:41:02","error_id":666}

We don't have any parameter to update media category.

**Comment by brianhorakh** [comment]
Sean - currently the media category is tied to the event, I don't believe it is possible for media to have a separate category (can you confirm)

Inder243 - I've fixed the VITA::update error.

**Comment by SeanGuerrero** [comment]
Yes, the media within a certain event can not have different or separate categories.  They are all tied to that one event and one category.

**Comment by inder243** [comment]
Please clarify below points:-
1) You mean to say, while editing any media of particular event, we can only change its caption ?

2) Currently we are not getting media caption in eventlist api, please provide "caption" param associated with each media in eventlist api. Please add  it.

**Comment by brianhorakh** [comment]
It is not possible (at the moment) to replace the media, and I would strongly argue that is a mistake since it could lead to some odd/questionable and difficult to police behaviors.  
E.x. user bumps photo A "fluffy kitten playing on road", which is then edited and replaced with photo B "lemon party" (don't google it)
I think the physical media should only be able to be removed, not modified once uploaded.

EventList now supports 'caption'

**Comment by SeanGuerrero** [comment]
To confirm, you will not be able to replace photo. A user will only be able to edit caption for the media that they uploaded.

**Comment by inder243** [comment]
Ok, Thanks will check/add and update you.

**Comment by SeanGuerrero** [comment]
Thanks Vakul!
